docs/man/docker-create.1.md

```diff
@@ -66,6 +67,12 @@ IMAGE [COMMAND] [ARG...]
 **--cpuset**=""
    CPUs in which to allow execution (0-3, 0,1)
 
+**--dev**=host
+   Use the systems /dev inside of the container
+     **host**: use the host's /dev directory inside the container.
+     No devices will be mounted on the /dev or any of its subdirs, from inside of the container
```

Comment:
What I was poorly trying to say was --dev=host, would also indicate that /dev/shm and /dev/mqueue would also be shared from host.  

docker currently mounts /dev, /dev/shm, and /dev/mqueue with tmpfs I believe.